documentation/modules/known-issues-2-7.adoc

```diff
@@ -16,9 +16,10 @@ include::snip_secure_boot_issue.adoc[]
 
 include::snip_measured_boot_windows_vm.adoc[]
 
+//do you want me to label this as resolved in 2.7.5 or delete as now included in resolved issues
 .Migration of a VM with Secure Boot enabled VM migration results in a VM with Secure Boot disabled
 
-When migrating a virtual machine (VM) with *Secure Boot* enabled, the VM has *Secure Boot* as `disabled` after being migrated. link:https://issues.redhat.com/browse/MTV-1632[(MTV-1632)]
+When migrating a virtual machine (VM) with *Secure Boot* enabled, the VM has *Secure Boot* as `disabled` after being migrated. This issue has been resolved in {project-short} 2.7.5. link:https://issues.redhat.com/browse/MTV-1632[(MTV-1632)]
```

Comment:
Tihs is actually a known issue. https://issues.redhat.com/browse/MTV-1632 is the tracker for the bug.

tldr: SecureBoot appears to be disabled, due to the bug above, but in reality it is enabled.